parla/task_runtime.py

```diff
@@ -783,51 +788,6 @@ def incr_active_tasks(self):
     def decr_active_tasks(self):
         self.scheduler.decr_active_tasks()
 
-    def estimated_queue_depth(self):
-        """Return the current estimated depth of this workers local queue.
-
-        This should be considered immediately stale and may in fact be slightly wrong (+/- 1 element) w.r.t. to any
-        real value of the queue depth. These limitations are to allow high-performance queue implementations that
-        don't provide an atomic length operation.
-        """
-        # Assume this will not FAIL due to concurrent access, slightly incorrect results are not an issue.
-        return len(self._queue)
-
-    def _pop_task(self):
-        """Pop a task from the queue head.
-        """
-        with self._monitor:
-            while True:
-                try:
-                    if self._should_run:
-                        logger.debug("Getting a task: %r", self)
-                        return self._queue.pop()
-                    else:
-                        return None
-                except IndexError:
-                    logger.debug("Blocking for a task: %r (%s)", self, self._monitor)
-                    print("[WorkerThread ", self.index,"] Blocking for a task.", sep='')
-                    self._monitor.wait()
-                    print("[WorkerThread ", self.index,"] Waking up.", sep='')
-
-    def steal_task_nonblocking(self):
-        """Pop a task from the queue tail.
-
-        :return: The task for None if no task was available to steal.
-        """
-        with self._monitor:
-            try:
-                return self._queue.popleft()
-            except IndexError:
-                return None
-
-    def _push_task(self, task):
-        """Push a local task on the queue head.
-        """
-        with self._monitor:
-            self._queue.append(task)
-            self._monitor.notify()
-
```

Comment:
deleted a crapton of code we no longer need since WorkerThreads don't need queues